usr/share/rear/restore/FDRUPSTREAM/default/260_copy_log_and_report.sh

```diff
@@ -1,14 +1,59 @@
-# upstream.log and upstream.rpt on the ReaR system contains valuable 
+# upstream.log and upstream.rpt on the ReaR system contain valuable 
 # information about the disaster recovery, so we archive *.log and *.rpt
 # to the restored system.
 
-EXTENSION="restore_$( date +%F_%T )"
+PREFIX="rear_$( date +%F_%T_%N )"
+SERVICENAME=( $(ps -ef | grep uscmd1 | grep servicename | grep -Po 'servicename=\K[^"]+') )
+REARLOGPATH="$FDRUPSTREAM_DATA_PATH/rear/logs"
+
+# If SERVICENAME is empty, then FDR/Upstream is not running and we won't 
+# be able to find the log files.
+if [[ ! ${SERVICENAME[@]} ]]; then 
+	echo
+	LogPrintError "***************"
+	LogPrintError "***************"
+	LogPrintError "No FDR/Upstream services were detected.  Unable to automatically copy logs and reports to the recovered system."
+	LogPrintError "Before rebooting, copy any necessary logs and reports from $FDRUPSTREAM_DATA_PATH into the $TARGET_FS_ROOT file tree."
+	LogPrintError "***************"
+	LogPrintError "***************"
+	echo
+	exit 1
```

Comment:
@rowens275 
please don't do an error exit via a plain `exit 1` in a script
but use the `Error` function that runs all the ReaR termination functions,
see usr/share/rear/lib/_input-output-functions.sh what the `Error` function does.

FYI
with
```
LogPrintError "Error exit 1"
exit 1
```
it looks this way
```
Error exit 1
rear mkrescue failed, check /root/rear.github.master/var/log/rear/rear-g243.log for details
```
while in contrast with
```
Error "Error exit 1"
```
it looks that way
```
ERROR: Error exit 1
Some latest log messages since the last called script 030_update_recovery_system.sh:
  2019-11-11 12:59:23.131229840 Including init/default/030_update_recovery_system.sh
Aborting due to an error, check /root/rear.github.master/var/log/rear/rear-g243.log for details
Terminated
```

Reply:
Done.  Thanks for the tip.